### ISSUE TITLE
Question about IMERG unit

### ISSUE THREAD

**Issue by binfenhuayun** [bennyistanto/landslide-rainfall#1]
I am very confused with the unit of IMERG data, according to the documetation, the unit of the variable "precipitationCal" is "mm/hr", so I am not very sure, the values can be summed simply, otherwise, the final data should be divided with 2?

**Comment by bennyistanto** [comment]
Since the unit is mm/hr, if you have a value of, say, 2 mm/hr, it means that in one hour, 2 mm of rain is expected to fall. For a half-hour period, this rate implies that 1 mm of rain would fall (since half the time, half the rain).

When you sum these half-hourly values over a full day, you're effectively adding up all the rain that fell in each half-hour segment throughout the day. There's no need to divide the result by 2 because each value already represents the amount of precipitation for that half-hour period, not an hourly rate that needs to be averaged out over the day.

By adding up all 48 half-hourly measurements for a day, you get the total daily precipitation in mm. This total does not need to be adjusted or divided by any number because the original measurements already account for the half-hourly periods.

So, in summary, to calculate daily precipitation totals from IMERG half-hourly data, you simply sum all the `precipitationCal` values across the 24-hour period. There's no need to divide the sum by 2 or make any other adjustments because each value directly contributes to the daily total based on the half-hour period it represents.

**Comment by binfenhuayun** [comment]
> Since the unit is mm/hr, if you have a value of, say, 2 mm/hr, it means that in one hour, 2 mm of rain is expected to fall. For a half-hour period, this rate implies that 1 mm of rain would fall (since half the time, half the rain).
> 
> When you sum these half-hourly values over a full day, you're effectively adding up all the rain that fell in each half-hour segment throughout the day. There's no need to divide the result by 2 because each value already represents the amount of precipitation for that half-hour period, not an hourly rate that needs to be averaged out over the day.
> 
> By adding up all 48 half-hourly measurements for a day, you get the total daily precipitation in mm. This total does not need to be adjusted or divided by any number because the original measurements already account for the half-hourly periods.
> 
> So, in summary, to calculate daily precipitation totals from IMERG half-hourly data, you simply sum all the `precipitationCal` values across the 24-hour period. There's no need to divide the sum by 2 or make any other adjustments because each value directly contributes to the daily total based on the half-hour period it represents.
So, for daily IMERG data, values have already been corrected to precipitation in a full day, which means I don't have to make them multiplying with 24, is that what you mean?
